src/modules/75-cf/components/FlagActivation/FlagActivationDetails.tsx

```diff
@@ -198,39 +197,18 @@ const FlagActivationDetails: React.FC<FlagActivationDetailsProps> = props => {
       )}
 
       <Container>
-        <Heading
-          style={{
-            fontWeight: 600,
-            fontSize: '16px',
-            lineHeight: '22px',
-            color: '#1C1C28',
-            marginBottom: 'var(--spacing-small)'
-          }}
-        >
-          {featureFlag.name}
-          {featureFlag.archived && (
-            <Text inline color={Color.GREY_400} padding={{ left: 'xsmall' }} font={{ size: 'small' }}>
-              ({getString('cf.shared.archived')})
-            </Text>
-          )}
+        <Heading>
+          <Text lineClamp={1} color={Color.BLACK} font={{ variation: FontVariation.H4 }}>
+            {featureFlag.name}
+          </Text>
         </Heading>
```

Comment:
Instead of a `p` tag in the heading, can we just shift this to the heading itself?
```suggestion
        <Heading
          level={4} // is this the right level?
          lineClamp={1}
          color={Color.BLACK}
          font={{ variation: FontVariation.H4 }}
        >
          {featureFlag.name}
        </Heading>
```

Reply:
nope - lineClamp exist as a prop on Heading

Reply:
Doh... been meaning to contribute that to the `Heading` component (really `Heading` aught to be a `Text` component with a different tag and the level.

Anyway, the `Heading` needs a level and setting as `inline` will give you a `span` instead of a `p`.
```suggestion
        <Heading level={4}>
          <Text inline lineClamp={1} color={Color.BLACK} font={{ variation: FontVariation.H4 }}>
            {featureFlag.name}
          </Text>
        </Heading>
```

The markup in NGUI leaves something to be desired.

Reply:
nice